sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -843,16 +847,6 @@ public EventDirectorySteps(
               getCheckboxByUUID(apiState.getCreatedEvent().getUuid()));
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
-
-    When(
```

Comment:
why removed? might be used in other tests

Reply:
was removing duplicates and this was deleted by mistake i added it back